deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -357,6 +357,21 @@ def callback_detect_file_deleted_event(line):
     return None
 
 
+def callback_detect_file_more_changes(line):
+    """ Callback that detects if a line in a log contains 'More changes' in content_changes.
+
+    Args:
+        line (String): string line to be checked by callback in FileMonitor.
+
+    Returns:
+        returns JSON string from log.
+    """
+    json_event = callback_detect_event(line)
+    if json_event is not None and 'content_changes' in json_event['data']:
+        if 'More changes' in json_event['data']['content_changes']:
+            return json_event
+

```

Comment:
This could be more generic, but would only serve for fields inside the data level of the json event. If the value we are looking for is nested inside a value, we would need another specific callback.

A workaround would be to search for the key inside the JSON, but this enforces the format and that the key is at the nesting level that it is expected.